packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -1,5 +1,7 @@
 <template>
   <div
+    v-click-outside="closeHandler"
+    :aria-expanded="open ? 'true' : 'false'"
```

Comment:
```suggestion
    :aria-expanded="open.toString()"
```
wdyt about this solution ;)?